pkgs/development/web/htmlbeautifier/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, bundlerApp, ruby }:
+
+bundlerApp {
+  inherit ruby;
+  pname = "htmlbeautifier";
+  gemfile = ./Gemfile;
+  lockfile = ./Gemfile.lock;
+  gemset = ./gemset.nix;
+  exes = [ "htmlbeautifier" ];
+
+  # toString prevents the update script from being copied into the nix store
+  passthru.updateScript = toString ./update;
```

Comment:
Question: Do all bundlerApp packages use such an update script? The script it self seems trivial that I doubt there is no generic way of doing so. It seems there is _something_ related to that according to https://nixos.org/nixpkgs/manual/#sec-language-ruby

Reply:
I found similar ones in [cocoapods](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/mobile/cocoapods/update) but I see some that don't have it, e.g [ruby-zoom](https://github.com/NixOS/nixpkgs/tree/master/pkgs/tools/text/ruby-zoom) and [solargraph](https://github.com/NixOS/nixpkgs/tree/master/pkgs/development/ruby-modules/solargraph).

Do you think I should remove it?

Reply:
I think cocoapods' update script is a just a bit "non-trivial" because it also handles a Gemfile-beta file for as far as I can see.

Can you try:

```nix
  passthru.updateScript = bundlerUpdateScript "htmlbeautifier";
```

You should be able to check if this works with:

```sh
nix-shell maintainers/scripts/update.nix --argstr package htmlbeautifier
```

See https://nixos.org/nixpkgs/manual/#var-passthru-updateScript

Reply:
@doronbehar ah amazing! thanks for clarifying. I changed it as you suggested and it seems to work.

Reply:
> Do all bundlerApp packages use such an update script?

Most of "simple ruby packages" do, since: https://github.com/NixOS/nixpkgs/pull/64822.